README.md

```diff
@@ -75,7 +75,7 @@ _Github PR reporter_
 <!-- table-of-contents-start -->
 ## Table of Contents
 
-- [MegaLinter, by](#megalinter-by-)
+- [MegaLinter, by ](#megalinter-by-)
```

Comment:
```suggestion
- [MegaLinter, by](#megalinter-by-)
```
Was this change intended or automatically changed?

Reply:
Looks like my TOC generator is slightly different than yours. Not sure why it added the space when I saved, I'll remove it manually.

FYI there's a second commit coming with the bulk of the changes. I wanted to make sure the first commit worked first (see OP for details).